pwem/xmipp-ghost/xmippLib.py

```diff
@@ -1061,14 +1061,27 @@ def MetaData():
 
 MetaDataInfo = None
 
-def Image():
-    pass
+HEADER = None
 
+class Image:
```

Comment:
We should think in the more general "backend" approach...but I'm fine with the change in the meantime.